atom/browser/lib/bluetooth_chooser.cc

```diff
@@ -31,11 +31,10 @@ const int kMaxScanRetries = 5;
 void OnDeviceChosen(
     const content::BluetoothChooser::EventHandler& handler,
     const std::string& device_id) {
-  if (device_id.empty()) {
+  if (device_id.empty())
```

Comment:
Our coding style does not enforce whether to have `{` for a simple case like this.